packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -125,6 +125,11 @@ type BaseBoxVisualProps = MakeObjectResponsive<
       | BackgroundColorString<'surface'>
       | BackgroundColorString<'action'>
       | (string & Record<never, never>);
+    backgroundImage: CSSObject['backgroundImage'];
```

Comment:
there is `PickByCSSPlatform` type below from line 150. Can we add these prop there? they will handle cross-platform and make it `undefined` on React Native so they won't need warnings.